dev/ci/docker/README.md

```diff
@@ -4,10 +4,21 @@ This directory provides Docker images to be used by Coq's CI. The
 images do support Docker autobuild on `hub.docker.com` and Gitlab's
 private registry.
 
-Gitlab CI will build and tag a Docker by default for every job if the
-`SKIP_DOCKER` variable is not set to `false`. In Coq's CI, this
-variable is usually set to `false` indeed to avoid booting a useless
-job.
+Gitlab CI will build and tag a Docker when running a pipeline if either of the
+conditions below is met:
+- If an image with the same name is not available in the
+  [Gitlab container registry](https://gitlab.com/coq/coq/container_registry)
+- If the `SKIP_DOCKER` variable is not set to `false`. In Coq's default CI, this
+  variable is set to `true` precisely to avoid running a lengthy, redundant job.
```

Comment:
No, the docker image is built if `SKIP_DOCKER` is not `true` (false is just one example of not `true`, unset is another), and the registry is used as cache so if a same-named image is there the "build" is a noop

Reply:
The not-false phrasing is not mine, it was like that before.